onnxjs/tests/test_cases.json

```diff
@@ -1,194 +1,242 @@
 [
     {
         "text": "This is terrible",
-        "isnegative": "1"
+        "isnegative": "1",
+        "importance": "1"
     },
     {
         "text": "You are a failure",
-        "isnegative": "1"
+        "isnegative": "1",
+        "importance": "1"
     },
     {
         "text": "This code is really bad",
-        "isnegative": "1"
+        "isnegative": "1",
+        "importance": "1"
     },
     {
         "text": "There are test failures",
-        "isnegative": "0"
+        "isnegative": "0",
+        "importance": "1"
```

Comment:
I don't think these lines in `test_cases.json` will do anything.

I think we need to figure out what to put here:

https://github.com/jonathanpeppers/inclusive-code-reviews-ml/blob/e46e2cc81f2d2c13e41c7b06c559d5c31044ede1/onnxjs/tests/onnx.ts#L15-L18

I tried something like `importance: new ort.Tensor('float32', [1,1]),`, but it's not quite right:
```
Error: Invalid rank for input: importance Got: 1 Expected: 2 Please fix either the inputs or the model.
```